### PR TITLE
wordy: implement "error" object

### PR DESCRIPTION
**Pull request by rpottsoh**
per #1311 proper error handling object incorporated into last two test cases.

### CODE REVIEW

File: exercises/wordy/canonical-data.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "wordy",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "comments": [
     "The tests that expect 'false' should be implemented to raise",
     "an error, or indicate a failure. Implement this in a way that",
```
Comment (Insti) [inline exercises/wordy/canonical-data.json]: This comment should probably be modified.

File: exercises/wordy/canonical-data.json [hunk 2]
```diff
@@ -125,15 +125,15 @@
       "input": {
         "question": "What is 52 cubed?"
       },
-      "expected": false
+      "expected": {"error": "Invalid Problem"}
```
Comment (ErikSchierboom) [inline exercises/wordy/canonical-data.json]: Maybe change this to `Invalid problem` (lowercase p)?

File: exercises/wordy/canonical-data.json [hunk 3]
```diff
@@ -125,15 +125,15 @@
...
     },
     {
       "description": "Non math question",
       "property": "answer",
       "input": {
         "question": "Who is the President of the United States?"
       },
-      "expected": false
+      "expected": {"error": "Invalid Problem"}
```
Comment (ErikSchierboom) [inline exercises/wordy/canonical-data.json]: Maybe change this to `Invalid problem` (lowercase p)?

File: exercises/wordy/canonical-data.json [hunk 4]
```diff
@@ -125,15 +125,15 @@
       "input": {
         "question": "What is 52 cubed?"
       },
-      "expected": false
+      "expected": {"error": "Invalid problem"}
```
Comment (Insti) [inline exercises/wordy/canonical-data.json]: This is OK, but the error message could probably be worded more helpfully.

**Pull request merged**